app/constant.ts

```diff
@@ -68,3 +68,4 @@ export const vaults: DApp[] = [
 ];
 
 export const pricePerNBTCInSUI = 25000;
```

Comment:
Write constant value in all capital letters.

Reply:
Done